ui/app/templates/vault/cluster/recovery.hbs

```diff
@@ -1,16 +0,0 @@
-{{!
-  Copyright (c) HashiCorp, Inc.
-  SPDX-License-Identifier: BUSL-1.1
-}}
-
-<Recovery>
```

Comment:
I think having a `recovery.hbs` route template or `recovery/index.hbs` would be useful if we had some sort of recovery dashboard or a shared header that all the child routes needed to render. However, the "recovery.load" route renders a different header so I think we can remove this all together.